runtime/dali/src/lib.rs

```diff
@@ -1148,6 +1192,7 @@ construct_runtime!(
 		Pablo: pablo::{Pallet, Call, Storage, Event<T>} = 65,
 		DexRouter: dex_router::{Pallet, Call, Storage, Event<T>} = 66,
 		StakingRewards: pallet_staking_rewards::{Pallet, Call, Storage, Event<T>} = 67,
+		Fnft: pallet_fnft::{Pallet, Storage, Event<T>} = 68,
```

Comment:
This can be simplified (can also apply to other pallets in the `construct_runtime!` macro)
```suggestion
		Fnft: pallet_fnft = 68,
```

Reply:
For now will not change just to keep things consistent. We can make a separate PR to fix this for the runtime